msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md

```diff
@@ -0,0 +1,158 @@
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: surbhigupta
+---
+
+# Manage your apps in Developer Portal
+
+After you create or upload your app, you can manage your apps in Developer Portal with the following:
+
+* [Overview](#overview)
+* [Configure](#configure)
+* [Advanced](#advanced)
+* [Publish](#publish)
+
+## Overview
+
+In the **Overview** section, you can see the following components to manage your app:
+
+* Dashboard
+
+  * In the **Dashboard** under **Overview** section, you can see the following components for your app:
+    * **Teams store validation**: App validation tool checks your app package against the test cases Microsoft uses when reviewing your app.
+    * **Announcement**: Latest updates of your apps on Developer Portal for Teams
+    * **Active users (Preview)**: Shows you the active user count
+    * **Basic information**: Shows you the App ID, Version, Manifest version, and so on.
+
+    :::image type="content" source="../../assets/images/tdp/dashboard-page.PNG" alt-text="The screenshot is an example that shows the Overview page of the app you created in Developer Portal for Teams.":::
+
+* Analytics
+
+    In the **Analytics** page under **Overview** section, you can see the total number of active users for your app. For more information, see [Analyze your app's usage](analyze-your-apps-usage-in-developer-portal.md).
+
+## Configure
+
+To install and render your app in Teams, you must include a set of configurations that Teams recognizes. To upload your apps in Teams, you need to have app manifest, which contains all the app details to display your app in Teams. This can be achieved with the help of components and tools that are available in Developer Portal.
+
+In the **Configure** section, you can see the following components to manage and access your app:
+
+* **Basic information**: This section shows and allows you to edit the App name, App ID, Descriptions, Version, Developer Information, App URLs, Application (client) ID, and Microsoft Partner Network ID.
+* **Branding**: This page shows the app icon details.
+* **App features**: You can add the following features to your app:
+  * Personal app
+  * Bot
+  * Connector
+  * Scene
+  * Group and channel app
+  * Messaging extension
+  * Meeting extension
+  * Activity feed notification
+* **Permissions**: This section allows you to give Device permissions, Team permissions, Chat or Meeting permissions, and User Permissions for your app.
+* **Single sign-on**: You can configure your app to authenticate users with single sign-on (SSO).
+* **Languages**: You can set up or change the language of your app.
+* **Domain**: You can add the domains to load your apps in the Teams client (For example: *.example.com).
+
+:::image type="content" source="../../assets/images/tdp/configure.png" alt-text="The screenshot is an example that shows how to configure features to manage and access your app in Developer Portal.":::
+
+## Advanced
```

Comment:
Advanced what?

Reply:
UI component under Apps page to manage apps in developer portal, and it is explained in the start.